core/core.go

```diff
@@ -140,17 +164,23 @@ func GenerateConfigFromFile(app *app.App, env *app.Environment, options *Generat
 		configFileName = envConfigFileName
 	}
 
-	if !app.HasMatch(configFileName) {
+	// always assume config file path is relative to the app source directory
+	// https://github.com/railwayapp/railpack/pull/226
```

Comment:
I don't think we need to include a link to the issue/pr for every change like this